share/man/man4/u3g.4

```diff
@@ -70,11 +90,12 @@ Qualcomm Inc. CDMA MSM
 .It
 Qualcomm Inc. GOBI 1000, 2000 and 3000 devices with MDM1000 or MDM2000 chipsets
 .It
-QUECTEL BGX, ECX, EGX, EMX, EPX, RGX series.
+QUECTEL BGX, ECX, EGX, EMX, EPX, RGX series
 .It
-Quectel EM160R (see CAVEATS)
+Quectel EM160R
+.Pq see Sx CAVEATS
 .It
-Huawei B190, E180v, E220, E3372, E3372v153, E5573Cs322, ('<Huawei Mobile>')
+Huawei B190, E180v, E220, E3372, E3372v153, E5573Cs322, etc.
```

Comment:
Does this change lose information? Is Huawei Mobile just a generic company, or some specific branding these are sold under?

Reply:
I'd not change it, please put it back.